README.md

```diff
@@ -112,15 +112,6 @@ the connection cannot do anything the user themselves couldn't. Other
 forms of authentication can be built on top of the JWT primitive. See
 the docs for more information.
 
-Since PostgreSQL 9.5 supports true [row-level
-security](http://www.postgresql.org/docs/9.5/static/ddl-rowsecurity.html).
-In previous versions it can be simulated with triggers and
-security-barrier views. Because the possible queries to the database
-are limited to certain templates using
-[leakproof](http://blog.2ndquadrant.com/how-do-postgresql-security_barrier-views-work/)
-functions, the trigger workaround does not compromise row-level
```

Comment:
I don't think we have RLS mentioned elsewhere, this paragraph could be included back into the README.

Reply:
Nevermind, I think this should be added in the docs instead.

Reply:
At first I removed all the parts about the trigger workaround, because that's not relevant anymore when we drop pg 9.5. But then.. nothing really remained anymore, except for "PostgreSQL supports true row-level security." :D

Would be great to add more about that in the docs, yes.